src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -358,3 +358,17 @@ def get_available_operations(parsed_mri: ParsedMRI) -> Sequence[str]:
     else:
         entity_key = get_entity_key_from_entity_type(parsed_mri.entity, True).value
         return AVAILABLE_GENERIC_OPERATIONS[entity_key]
+
+
+def extract_use_case_id(mri: str) -> UseCaseID:
```

Comment:
Moved the function here since there was a circular dependency otherwise and it makes more sense to have MRI specific functions in this module.